R/cell_count_plot.R

```diff
@@ -30,6 +30,8 @@ CellCountPlot.data.frame <- function(
   color_by,
   show_count = TRUE,
   flip_axes = FALSE,
+  as_frequency = TRUE,
```

Comment:
`as_frequency` and `stack` needs to be documented using `@param`.

Reply:
The parameters are now documented in `generics.R`